extension/content.js

```diff
@@ -409,6 +409,13 @@ function fixSquashAndMergeTitle() {
 	});
 }
 
+function addTitleToEmojis() {
+	const emojis = $('.js-comment-body, #readme').find('g-emoji');
```

Comment:
It would cover all of the emojis including the ones in reactions to a comment/issue created. I figured we care only about the things the user can define, thus such selector. Good point on the readme one, totally forgot about other possibilities. Can change it to something like `$('. markdown-body').find('g-emoji')` which would target the Markdown documents, issues, PRs, comments. WDYT?